src/Client.cpp

```diff
@@ -402,7 +401,7 @@ void CClient::ReadLine(const CString& sData) {
 		sPatterns.Split(" ", vsChans, false, "", "", true, true);
 
 		set<CChan*> sChans;
-		for (const CString& sChan : vsChans) {
+		for (auto sChan : vsChans) {
```

Comment:
`for (const CString& sChan : vsChans)` was not modern enough? And it does not make a copy